Split/Api/SplitClientConfig.swift

```diff
@@ -231,6 +221,19 @@ public class SplitClientConfig: NSObject {
     ///
     @objc public var synchronizeInBackground = false
 
+    /// Indicates if events and impressions are tracked and sent to BE.
+    /// @param mode Values:<br>
+    ///     UNKNOWN: Impressions and events are tracked in memory. They are lost on destroy
+    ///     GRANTED: Impressions and events are tracked and sent. They remains stored when the SDK is destroyed.
+    ///     many times impressions are posted. If previous value was UNKNOWN, tracked data is persisted.
```

Comment:
Just checking; is the `many times impressions are posted` sentence OK?

Reply:
mmm not really :S Thanks @gthea! will change that in next PR